.github/workflows/metadata-update.yml

```diff
@@ -0,0 +1,85 @@
+name: Metadata Update
+
+on:
+  workflow_dispatch:  # allow this to be manually triggered
+  schedule:
+    - cron: "00 1 * * *" # daily at 1:00 UTC
+
+permissions:
+  contents: read
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Free disk space
+        run: .github/scripts/gha-free-disk-space.sh
+
+      - name: Set up JDK for running Gradle
+        uses: actions/setup-java@c5195efecf7bdfc987ee8bae7a71cb8b11521c00 # v4.7.1
+        with:
+          distribution: temurin
+          java-version-file: .java-version
+
+      - name: Set up gradle
+        uses: gradle/actions/setup-gradle@ac638b010cf58a27ee6c972d7336334ccaf61c96 # v4.4.1
+        with:
+          cache-read-only: true
+
+      - name: Check out main branch
+        run: git fetch origin main && git checkout main && git pull origin main
+
+      - name: Collect telemetry
+        run: ./instrumentation-docs/ci-collect.sh
+
+      - name: Run documentation analyzer
+        run: ./gradlew :instrumentation-docs:runAnalysis
+
+      - name: Check for diff
+        id: diffcheck
+        run: |
+          git add docs/instrumentation-list.yaml
+          if ! git diff --cached --quiet; then
+            echo "::set-output name=has_diff::true"
+          else
+            echo "::set-output name=has_diff::false"
+          fi
+
+      - uses: actions/create-github-app-token@df432ceedc7162793a195dd1713ff69aefc7379e # v2.0.6
+        if: steps.diffcheck.outputs.has_diff == 'true'
+        id: otelbot-token
+        with:
+          app-id: 1295839
```

Comment:
do I need a different app-id for this? (I copied this from https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/.github/workflows/auto-update-pull-request.yml#L36)